app/views/course/assessment/question/scribing/edit.json.jbuilder

```diff
@@ -0,0 +1,8 @@
+json.form_data do
```

Comment:
Frontend code should take care of this itself.  the things inside API folder should know all these ( i.e. what HTTP verb to use and what's the auth_token ).